src/views/waterUse/WaterUse.vue

```diff
@@ -45,19 +59,44 @@
                 title: process.env.VUE_APP_TITLE,
                 titleSuffix: process.env.VUE_APP_TITLE_SUFFIX,
                 featureName: 'Water Use',
-                developmentTier: process.env.VUE_APP_TIER
+                developmentTier: process.env.VUE_APP_TIER,
+                mapStyle: mapStyles.style,
+                container: 'map',
+                zoom: 2,
+                minZoom: 2,
+                maxZoom: 5.99,
+                center: [-95.7129, 37.0902],
+                pitch: 0, // tips the map from 0 to 60 degrees
+                bearing: 0, // starting rotation of the map from 0 to 360
+                hoveredHRUId: null,
+                maxBounds: [[-168.534393,-4.371744], [-19.832382,71.687625]], // The coordinates needed to make a bounding box for the continental United States.
+                isLoading: true,
+            }
+        },
+        methods: {
+            runGoogleAnalytics(eventName, action, label) {
+                this.$ga.set({ dimension2: Date.now() });
+                this.$ga.event(eventName, action, label);
+            },
+            onMapLoaded(event) {
+                this.$store.map = event.map; // The 'event' gives us access to the map as an object but only after the map has loaded. Once we have that, we add the map object to the Vuex store
+                const map = this.$store.map;
+                let googleAnalytics = this.runGoogleAnalytics; // We need to get the global Google Analytics (GA) plugin object 'this.$ga' into this scope, so let's make a local variable and assign our GA event tracking method to that.
+                map.resize(); //This solves the mysterious whitespace by resizing the map to the correct size.
+                map.touchZoomRotate.enable(); // Allow users to pinch to zoom on touch devices.
+                map.touchZoomRotate.disableRotation(); // Disable the rotation functionality, but keep pinch to zoom.
+                map.fitBounds([[-125.3321, 23.8991], [-65.7421, 49.4325]]); // Once map is loaded, zoom in a bit more so that the map neatly fills the screen.
+                setTimeout(() => { this.isLoading = false; }, 200);// Set a timeout to make sure the fitbounds action is completely done before loading screen fades away.
+                map.on('zoomend', this.addZoomLevelIndicator); // Add the current zoom level display. The zoom level should only show in 'development' versions of the application.
             }
         }
     }
 </script>
 
 <style scoped lang="scss">
 #water-use-container {
-  border: solid black;
```

Comment:
This section provides the tab effect. So, although it is hard to see with the dark blue of the tab, it gives the effect that the current tab is part of the page and the other tabs are separated from the page with a thin back line. We are going to talk about the styling of the tabs at the Crit on Thursday, so I would leave it in for now.